src/main/java/edu/pw/eiti/pik/user/UserController.java

```diff
@@ -47,6 +57,25 @@ UserDto me() {
         return userMapper.toDto(user);
     }
 
+    @ApiOperation(value = "Get top n users by name who have given role")
+    @ApiResponses({
+    	@ApiResponse(code = 200, message = "Always")
+    })
+    @GetMapping(path = "/user/{authority}/{name}/{n}")
```

Comment:
Name 'n' is not acceptable. Name should say what it is/does

Reply:
I believe that n is widely-recognized name for "number (of)" but will change that